engine/src/flutter/lib/ui/painting.dart

```diff
@@ -193,6 +234,13 @@ class Color {
   /// * Bits 16-23 are the red value.
   /// * Bits 8-15 are the green value.
   /// * Bits 0-7 are the blue value.
+  ///
+  /// > [!WARNING]
```

Comment:
This warning would be more useful on the non-deprecated `toARGB32` method rather than here. In fact, maybe we should turn this dartdoc into a stub that simply points to `toARGB32`?

Reply:
Done.